### PR TITLE
Final fixes

### PR DESCRIPTION
**Pull request by p6te**
- add tooltips for token link and estimated balance
- add hover highlight for copy market id
- fix tokens animation on swap

### CODE REVIEW

**Review by zielvna — APPROVED** [review]

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
IMO, the tooltip for balance should be consistent, because it has been added in only one place. Therefore, it should also be added in other places - the entire app needs to be reviewed to find the remaining locations:
image

There is none here:
- Screenshot from 2024-08-08 04-38-14
- Screenshot from 2024-08-08 04-38-06


Additionally, the description should indicate what this balance means, because on "Add position" this value represents the wallet balance of selected token, while on "position details" this balance value represents the token value in the open position.

**Comment by zielvna** [comment]
image
there is hover effect on add new position page

File: src/components/Inputs/ExchangeAmountInput/ExchangeAmountInput.tsx [hunk 1]
```diff
@@ -184,11 +184,18 @@ export const AmountInput: React.FC<IProps> = ({
               priceLoading ? (
                 <img src={loadingAnimation} className={classes.loading} alt='loading' />
               ) : tokenPrice ? (
-                <>
+                <Tooltip
+                  enterTouchDelay={0}
+                  leaveTouchDelay={Number.MAX_SAFE_INTEGER}
+                  title='Your Estimated Balance in USD'
```
Comment (wojciech-cichocki) [inline src/components/Inputs/ExchangeAmountInput/ExchangeAmountInput.tsx:190]:
```suggestion
                  title='Estimated USD Value of the Selected Tokens in Your Wallet'
```

File: src/components/Inputs/DepositAmountInput/DepositAmountInput.tsx [hunk 2]
```diff
@@ -164,9 +164,18 @@ export const DepositAmountInput: React.FC<IProps> = ({
               priceLoading ? (
                 <img src={loadingAnimation} className={classes.loading} alt='loading' />
               ) : tokenPrice ? (
-                <Typography className={classes.caption2}>
-                  ~${formatNumber(usdBalance.toFixed(2))}
-                </Typography>
+                <Tooltip
+                  enterTouchDelay={0}
+                  leaveTouchDelay={Number.MAX_SAFE_INTEGER}
+                  title='Your Estimated Balance in USD'
```
Comment (wojciech-cichocki) [inline src/components/Inputs/DepositAmountInput/DepositAmountInput.tsx:170]:
```suggestion
                  title='Estimated USD Value of the Selected Tokens in Your Wallet'
```

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**